src/applications/claims-status/containers/ClaimsStatusApp.jsx

```diff
@@ -5,23 +5,47 @@ import PropTypes from 'prop-types';
 
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { RequiredLoginView } from 'platform/user/authorization/components/RequiredLoginView';
+
 import { setLastPage } from '../actions';
 import ClaimsAppealsUnavailable from '../components/ClaimsAppealsUnavailable';
+import { isLoadingFeatures } from '../selectors';
 
 // This needs to be a React component for RequiredLoginView to pass down
 // the isDataAvailable prop, which is only passed on failure.
-function AppContent({ children, isDataAvailable }) {
+function AppContent({ children, featureFlagsLoading, isDataAvailable }) {
   const canUseApp =
     isDataAvailable === true || typeof isDataAvailable === 'undefined';
+  const isAppReady = canUseApp && !featureFlagsLoading;
+
+  if (!isAppReady) {
```

Comment:
I think because my last comment was tied to this line it got marked as "out of date" with the last commit. It was:

> Since we added some logic to the component, I'm thinking we should probably augment or add a test for the CSTApp container. Thoughts?

Reply:
I added some unit tests, let me know what you think